code-challenges.js

```diff
@@ -22,6 +40,9 @@ var temp3 = 212
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+for(let i=0; i<myNumbers1.length; i++){
+  console.log(myNumbers1[i] * 5)
+}
```

Comment:
this is super close! the logic is all there but we want it to be wrapped in a function that will require a return instead of console.logging in the loop.